src/NHibernate.Test/NHSpecificTest/Logs/LogsFixture.cs

```diff
@@ -73,6 +74,7 @@ public TextLogSpy(string loggerName, string pattern)
 				};
 				loggerImpl = (Logger)LogManager.GetLogger(typeof(LogsFixture).Assembly, loggerName).Logger;
 				loggerImpl.AddAppender(appender);
+				previousLevel = loggerImpl.Level;
 				loggerImpl.Level = Level.All;
```

Comment:
This spy was used only for `NHibernate.SQL`, but that is already an heavily logging logger. Better restore its previous log level.